### DISCUSSION TITLE
How can I display a nested set in a table?

### DISCUSSION THREAD

**Discussion by Banditman74** [filamentphp/filament#8442] [Help]
Greetings to all!
Is it possible to somehow display the structure of a nested set in a table so that they form a tree structure? For example, as multiple catalog categories with subordinate categories.

**Comment by leandrocfe** [comment]
Isn't possible I think, but you can [group rows](https://filamentphp.com/docs/3.x/tables/grouping#grouping-rows-by-default)

**Reply by Banditman74** [reply]
Thanks for your feedback

**Reply by zipavlin** [reply]
@Banditman74 did you maybe manage to figure this out? I would need this to build a page-tree ie. nested pages in table.
